src/parse.rs

```diff
@@ -33,7 +33,11 @@ pub fn records_to_columns<S: ::std::hash::BuildHasher>(
             })),
             DataType::Enum => Column::with_data(records.iter_mut().fold(vec![], |mut col, v| {
                 let val: String = v.next().unwrap().iter().map(|&c| c as char).collect();
-                let enum_value = labels.get(&fid).unwrap().get(&val).map_or(0, |val| *val);
+                let enum_value = if let Some(map) = labels.get(&fid) {
```

Comment:
How about using [`unwrap_or_default`](https://doc.rust-lang.org/std/option/enum.Option.html#method.unwrap_or_default)?

Reply:
I overlooked that the `Option` is for `HashMap`. Please ignore my comment.